examples/cfq/models.py

```diff
@@ -78,8 +82,13 @@ def apply(self,
     scores = jnp.where(mask, scores, -jnp.inf)  # Using exp(-inf) = 0 below.
     scores = nn.softmax(scores, axis=-1)
 
-    context = jnp.sum(jnp.expand_dims(scores, 2) * values, axis=1)
-    return context, scores  # Shape: <float32>[batch_size, seq_len]
+    if batch_level:
+      # Shape: <float32>[batch_size, seq_len]
+      context = jnp.sum(jnp.expand_dims(scores, 2) * values, axis=1)
```

Comment:
The third dimension/axis is expanded instead of the second one (when the data is at batch level, the batch is at the first axis - 0, and the second axis - 1 becomes the third axis -2)